politeiawww/backend.go

```diff
@@ -255,6 +260,21 @@ func (b *backend) _convertPropFromInventoryRecord(r inventoryRecord) www.Proposa
 	return proposal
 }
 
+func getProposalStatusTimestamp(ir inventoryRecord) (int64, int64, int64) {
```

Comment:
Should this be `getProposalStatusTimestamps` since it is returning multiple timestamps?